lib/plugins/package/lib/packageService.js

```diff
@@ -36,10 +36,10 @@ module.exports = {
   getExcludes(exclude, excludeLayers) {
     const packageExcludes = this.serverless.service.package.exclude || [];
     // add local service plugins Path
-    const pluginsLocalPath = this.serverless.pluginManager.parsePluginsObject(
-      this.serverless.service.plugins
-    ).localPath;
-    const localPathExcludes = pluginsLocalPath ? [pluginsLocalPath] : [];
+    const localPluginPathExcludes = this.serverless.pluginManager.getLocalPluginsPaths()
```

Comment:
This part should not be changed afaik.

it's just that `localPathExcludes` should be enriched with paths that we'd group at `this.serverless.pluginManager.localPluginsPaths`

Reply:
@medikoo Perhaps, but I don't think there's any harm. We're still achieving the same thing, yes? "Give me all the paths for local plugins". This way, it's a straightforward request to the plugin manager. Does this affect anything? Am I missing something?

Reply:
The problem to me, is that we touch here logic we should keep. We also need to be careful with changes to not break anything, so instead of refactoring reshaping existing parts, we should rather leave them intact, and just add 2 cents which resolves those few more paths we need to cover in exclusion.

So my point is that the resolution of `const pluginsLocalPath` should remain the same (be left untouched), we still need that path in the form it was resolved. Let's just add logic which adds those extra paths to `localPathExcludes`, that's it.

Also, it's good to keep old handling of local path plugins with new handling as separate, as it might be that we will want to drop support for the old way, if we mix it, it'll be harder

Reply:
Another advantage of doing it this way: when we remove support for `.serverless_plugins` and `plugins.localPath`, as I've seen is planned, the only change will be to not add it to the `localPluginsPaths` array in the plugin manager. The packager does not have to know or care about `.serverless_plugins`, and its code will not change,

Reply:
> the only change will be to not add it to the localPluginsPaths array in the plugin manager. 

Why do you assume so? I think we need this collection to collect there all local plugins referenced by relative paths.

Reply:
Hmmm...I don't really understand.

But I see your point about not refactoring existing code (though I don't entirely agree😀). I'll make this change.